packages/api/src/hooks/useAvailableWallets.ts

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import useFetch from '../useFetch';
+import useWalletAccountsList from './useWalletAccountsList';
+import useAuthorize from './useAuthorize';
+import useCurrencyConfig from './useCurrencyConfig';
+
+const useAvailableWallets = () => {
+    const { data } = useAuthorize();
+    const { data: account_type_data, ...rest } = useFetch('get_account_types', {
+        payload: {
+            // This is temporary fix, will be changed by BE
+            company: data?.landing_company_name === 'virtual' ? 'svg' : data?.landing_company_name,
```

Comment:
We shouldn't rely on the account's landing_company since it could be changed while changing the active_wallet

Reply:
But we need to send landing_company as payload to the request